compiler/src/iree/compiler/Dialect/Flow/IR/test/tensor_folding.mlir

```diff
@@ -112,7 +112,7 @@ util.func public @reshapeNoOpDynamic(%arg0: tensor<4x?xf32>, %dim: index) -> ten
 
 // CHECK-LABEL: @reshapeDynamicDifferent
 util.func public @reshapeDynamicDifferent(%arg0: tensor<4x?xf32>, %dim0: index, %dim1: index) -> tensor<4x?xf32> {
-  // CHECK-NEXT: flow.tensor.reshape %arg0
+  // CHECK-NEXT: util.return %arg0 : tensor<4x?xf32>
```

Comment:
maybe add a new test with two values instead to ensure it's preserved

Reply:
Good point + checks that it doesnt get completely folded